test/test.googleauth.ts

```diff
@@ -810,6 +823,9 @@ describe('googleauth', () => {
   });
 
   it('getApplicationDefault should cache the credential when using GCE', async () => {
+    // tslint:disable-next-line no-any
+    sinon.stub(auth as any, 'getDefaultServiceProjectId').resolves();
```

Comment:
This is what `createGetProjectIdNock` is supposed to do.  Something is very wrong if we need this

Reply:
(here and below)

Reply:
Changed - please take a look!